tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -82,18 +82,135 @@ int main() {
     vformat_to(back_insert_iterator(output_string), locale::classic(), "{0} {0}", make_format_args((const char*) "f"));
     assert(output_string == "f f");
 
-    // TODO: enable these in _Write function PR for sv and bool
-    /*
+    // Test string_view
     output_string.clear();

```

Comment:
This is more of a style nitpick, feel free to ignore:

I found it much easier to reason about when I gave each and every test a descriptive string. Like `output_from_string_view`. It is equally long due to not needing to clear and i can see in the assert what is really tested

Reply:
I don't fully follow, can you elaborate? Do you have an example test with this style? I couldn't find anything with `output_from_string_view`.

Reply:
I would have written:

```cpp
string output_from_string_view;
vformat_to(back_insert_iterator(output_from_string_view), locale::classic(), "{}", make_format_args("f"sv));
assert(output_from_string_view== "f");
```

Reply:
Essentially the style I used in the constexpr vector / nstring tests

Reply:
Ah I see what you mean. My main concerns are coming up with good names and copy-paste errors.

Reply:
I think if you test something you already have a name. `fmt_double_infinity`

Copy & paste is indeed a problem

Reply:
eventually we'll probably move some of these tests to their own functions, and we can name them. For now I don't really care that much about the output string's name.